Engine/Annotations/HasExpressionAnnotation.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using OpenTap.Expressions;
+namespace OpenTap
+{
+    class HasExpressionAnnotation : IIconAnnotation, IEnabledAnnotation, IInteractiveIconAnnotation, IExpressionAnnotation, IOwnedAnnotation, IErrorAnnotation
```

Comment:
Not sure if we do this elsewhere, but i would highly suggest adding sealed to classes, if no classes inherit from them.
Makes calling methods on the class roughly 10x faster, and uses a locked down by default style, meaning you have fewer breaking changes that needs to be considered in the future.
```suggestion
    class HasExpressionAnnotation : IIconAnnotation, IEnabledAnnotation, IInteractiveIconAnnotation, IExpressionAnnotation, IOwnedAnnotation, IErrorAnnotation
```